src/static/css/content.less

```diff
@@ -279,6 +276,10 @@ figure {
 
```

Comment:
@Scotchester - Does the tint-swatch table convert to a single column like the secondary colors?